app/config.py

```diff
@@ -376,8 +376,9 @@ class Config(object):
     HC_EN_SERVICE_ID = os.getenv('HC_EN_SERVICE_ID', '')
     HC_FR_SERVICE_ID = os.getenv('HC_FR_SERVICE_ID', '')
     BULK_SEND_TEST_SERVICE_ID = os.getenv('BULK_SEND_TEST_SERVICE_ID', '')
-    CSV_MAX_ROWS = os.getenv('CSV_MAX_ROWS', 50000)
-    CSV_MAX_ROWS_BULK_SEND = os.getenv('CSV_MAX_ROWS_BULK_SEND', 100000)
+    CSV_MAX_ROWS = os.getenv('CSV_MAX_ROWS', 50_000)
+    CSV_MAX_ROWS_BULK_SEND = os.getenv('CSV_MAX_ROWS_BULK_SEND', 100_000)
+    CSV_BULK_REDIRECT_THRESHOLD = os.getenv('CSV_BULK_REDIRECT_THRESHOLD', 1_000)
```

Comment:
Think we will set that to a lower value in production. At 20-30 emails per second it already takes a good amount of time to send 500 emails so 1k seems too much

Reply:
Okay, I went the she specified number [in the Trello card](https://trello.com/c/rZEFQezH/472-route-notifications-sent-using-spreadsheets-to-the-bulk-queue). I am good with the suggestion.

Reply:
I spoke with Antoine offline and we resolved to lower the default to `200` entries in a CSV for the bulk queue redirection to activate.